### PR TITLE
nitunit: fix running last test of a module, it was previously ignored

### PR DESCRIPTION
**Pull request by xymus**
This is a very small commit, but it may reveal some previously ignored nitunits.

### CODE REVIEW

**Comment by xymus** [comment]
Raise an error in a previously untested nitunit of lib/github/cache.nit. The second assert fails, which would indicate that the cache did not work.

@Morriar , can you take a look? Everything looks good to me, it must be a detail.

**Comment by privat** [comment]
C'est le premier assert qui semble peter. Sans doute un cache reste d'un précédent test.

**Comment by privat** [comment]
+1 sinon

File: lib/github/cache.nit [hunk 1]
```diff
@@ -29,7 +29,7 @@
 # var name = "privat/nit"
```
Comment (Morriar) [inline lib/github/cache.nit]: You can add `api.clear_cache` at the beginning of the test to avoid the crash on the first assert.

My bad, sorry.

**Comment by Morriar** [comment]
+1, I never noticed this, pretty lucky that only one test is failing here :p

**Comment by xymus** [comment]
Sorry guys, but it is the second assert that crashes, at line 13: `assert api.has_cache(name)`. Look at the last commit where it is disabled. Clearing the cache doesn't fix it.

The cache do not work for some reason on my system and on pratchett. Configuring my system with an API key didn't solve the problem.

I would let the honnor to @Morriar to fix and reactivate the test.

**Pull request merged**